src/current/cockroachcloud/cloud-api.md

```diff
@@ -335,12 +478,71 @@ If the request was successful, the API will return detailed information about th
+
+{{site.data.alerts.callout_info}}
+The `spend_limit` field, which was deprecated in Serverless, is not supported on Basic or Standard. Instead, use `usage_limits`.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+The service account associated with the secret key must have the Cluster Administrator or Cluster Developer [role]({% link cockroachcloud/authorization.md %}#organization-user-roles).
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="curl"><strong>curl</strong></button>
+    <button class="filter-button page-level" data-scope="raw"><strong>Raw</strong></button>
+</div>
 
-## Set the maximum resource limits of a Serverless cluster
+<section class="filter-content" markdown="1" data-scope="curl">
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request PATCH \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id} \
+  --header 'Authorization: Bearer {secret_key}' \
+  --json '{"serverless":{"usage_limits":{"storage_mib_limit":"5242880","request_unit_limit":"50000000"}}}'
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="raw">
+
+{% include_cached copy-clipboard.html %}
+~~~ json
+{
+  "serverless": {
+    "usage_limits": {
+      "storage_mib_limit": "5242880",
+      "request_unit_limit": "100000000"
+    }
+  }
+}
+~~~
+
+</section>
+
+Where:
+
+  - `{cluster_id}` is the unique ID of this cluster.
+  {{site.data.alerts.callout_info}}
+  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-your-cluster.md %}).
+  {{site.data.alerts.end}}
+  - `{secret_key}` is the secret key for the service account.
+  - `storage_mib_limit` is the maximum number of Mebibytes of storage that the cluster can have at any time during the month. If this limit is exceeded, then the cluster is throttled, where only one SQL connection is allowed at a time, with the expectation that it is used to delete data to reduce storage usage. It is an error for this value to be zero.
+  - `request_unit_limit` is the maximum number of request units that the cluster can consume during the month. If this limit is exceeded, then the cluster is disabled until the limit is increased, or until the beginning of the next month when more request units are granted. It is an error for this to be zero.
 
-To set the maximum resource limits for a Serverless cluster, send a `PUT` request to the `/v1/clusters/{cluster_id}/spend-limit` endpoint.
+If the request was successful, the API returns the updated cluster details.
+
+For details about returned fields, refer to the [response example and schema](https://www.cockroachlabs.com/docs/api/cloud/v1.html#patch-/api/v1/clusters/-cluster_id-) in the API reference.
+
+## Change the number of provisioned vCPUs for a Standard cluster
```

Comment:
This might be a good place to mention that the number of provisioned vCPUs can be increased any number of times, but can only be decreased up to 3 times per week.

Reply:
Done. Thanks.